docs/code-review.md

```diff
@@ -0,0 +1,61 @@
+# Code Review Checklist
+
+Currently, all changes to the product must be reviewed by a [core committer](/contribute/getting-started/core-committers/#core-committers).
+
+<!-- * Documentation changes must be reviewed by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+  * Product managers may ask for reviews from [core committers](/contribute/getting-started/core-committers/#core-committers) and [QA testers](/contribute/getting-started/core-committers/#qa-testers) as required. -->
+
+If you are a community member seeking a review
+----------------------------------------------
+
+1. Submit your pull request.
+    * Follow the [contribution checklist](../contribution-checklist/).
+2. Wait for a reviewer to be assigned.
+    * Product managers are on the lookout for new pull requests and usually handle this for you automatically.
+    * If you have been working alongside a core committer, feel free to message them for help.
+    * When in doubt, ask for help in the [Focalboard](https://community.mattermost.com/core/channels/focalboard) channel on our community server.
+    * If you are still stuck, please message Chen Lim (@chenilim on GitHub).
+3. [Wait for a review](#if-you-are-awaiting-a-review).
+    * Expect some interaction with at least one reviewer within 5 business days (weekdays, Monday through Friday, excluding [statutory holidays](https://docs.mattermost.com/process/working-at-mattermost.html#holidays)).
+    * Keep in mind that core committers are geographically distributed around the world and likely in a different time zone than your own.
+    * If no interaction has occurred after 5 business days, please at-mention a reviewer with a comment on your PR.
+4. Make any necessary changes.
+    * If a reviewer requests changes, your pull request will disappear from their queue of reviews.
+    * Once you've addressed the concerns, please at-mention the reviewer with a comment on your PR.
```

Comment:
nit: Mix of `PR` and `pull request` on this page.  It could be argued that someone reading this knows they are the same, but it wouldn't hurt to write out the name.